IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -29,6 +29,9 @@
 #import "MSIDTelemetryEventStrings.h"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->